<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>progressBar_2</class>
 <widget class="QWidget" name="progressBar_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Downloder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="Download">
     <property name="text">
      <string>دانلود</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="txt_url"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="txt_dis"/>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="Browser">
     <property name="text">
      <string>مرورگر</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
